src/Apps/Sell/Components/BottomFormNavigation.tsx

```diff
@@ -180,10 +181,14 @@ const BottomFormNextButton = () => {
     }
   }, [value, clearValue, onAfterAuthCallback])
 
+  // for guest users make userPhone field required
+  const disableIfLoggedOut =
+    !isLoggedIn && step === "phone-number" && !(values as FormValues).userPhone
+
   return (
     <Button
       variant="primaryBlack"
-      disabled={!isValid || isSubmitting || loading}
+      disabled={!isValid || isSubmitting || loading || disableIfLoggedOut}
```

Comment:
Not sure if forcing the user to add a phone number is the best approach
Why didn't we make it required in the first place? Are there any legal concerns related to requesting a phone number from the user that doesn't have an account?